docs/automated_testing.md

```diff
@@ -991,38 +993,162 @@ uses: suffolkLITLab/ALKiln@releases/v4
 
 ```yml
-      - name: Set env vars
-        run: |
+env:
 ```
 
-4. Whenever you want to add a secret, add a new line under those in this format:
+4. Whenever you want to add a secret, add a new line under `env:` indented once as shown below:
 
 ```yml
-          echo "USER_PASSWORD=${{ secrets.USER_PASSWORD }}" >> $GITHUB_ENV
+  USER_PASSWORD: "${{ secrets.USER_PASSWORD }}"
 ```
 
 `USER_PASSWORD` is just a placeholder in our example. You can name your secrets whatever you want to. Make sure you use the same words as the GitHub secret you made.
 
 5. Write your Step and use the names of these secrets as the values.
 
+If you're not worried about keeping the information secure, you don't have to use a GitHub secret - you can just put the value straight into your workflow file like this:
+
+```yml
+  MAX_SECONDS_FOR_SERVER_RELOAD: "300"
+```
+
+ALKiln will still avoid printing this value and avoid taking screenshots when it is used. There is no way ALKiln can tell which environment variables have sensitive information and which ones are safe.
+
+All together, this section can look similar to this:
+
+```yml
+env:
+  USER_PASSWORD: "${{ secrets.USER_PASSWORD }}"
+  MAX_SECONDS_FOR_SERVER_RELOAD: "300"
+
+jobs:
+```
+
+
+## Your workflow file
+
+This whole section is very technical. Feel free to ask us questions.
+
+**Where is it?**
+
+Your ALKiln workflow file is in your repository. To find it, go to your `.github` folder, then open the `workflows` folder there. It was probably created when you ran the setup interview and it might be called "run_form_tests.yml" or "alkiln_tests.yml" or something similar.
+
+**What does it do?**
+
+Among other things, the workflow file:
+
+- Triggers the ALKiln tests when desired, like when you push new code to your package.
+- Gives ALKiln inputs that are needed to run your tests.
+- Optionally gives ALKiln other inputs and environment variables it can use.
+
+You can also use the whole suite of GitHub's workflow and action functionality to do other things, like creating issues when tests fail.
+
+### Required inputs
+
+The setup interview should have helped you create these required `inputs` and their values. They are in the `jobs:` section. They look something like this:
+
+```yml
+        with:
+          SERVER_URL: "${{ secrets.SERVER_URL }}"
+          DOCASSEMBLE_DEVELOPER_API_KEY: "${{ secrets.DOCASSEMBLE_DEVELOPER_API_KEY }}"
+```
+
+`SERVER_URL` is the url of the docassemble server the tests should run on.
 
-## Customizations
+`DOCASSEMBLE_DEVELOPER_API_KEY` is the API key that you created for the account on your server that will store the Project in the Playground while the tests are being run. You probably created this in the setup interview. Alternatively, your organization admin may have created it.
+
+We recommend keeping the API key a GitHub secret for security reasons, but the server url can be type in plainly. For example `SERVER_URL: "https://apps-test.suffolklitlab.com"`.
```

Comment:
IMO it's not best practice to use our real URLs in docs, but I'm kinda paranoid there 😅 
```suggestion
We recommend keeping the API key a GitHub secret for security reasons, but the server url can be type in plainly. For example `SERVER_URL: "https://apps-test.example.com"`.
```